tasklib/tests.py

```diff
@@ -818,7 +821,8 @@ def test_return_all_from_executed_command(self):
     def test_return_all_from_failed_executed_command(self):
         Task(self.tw, description="test task", tags=['test']).save()
         out, err, rc = self.tw.execute_command(['countinvalid'],
-            return_all=True, allow_failure=False)
+                                               return_all=True,
+                                               allow_failure=False)
```

Comment:
I'd probably opt for each argument on a new line here:

```python
out, err, rc = self.tw.execute_command(
    ['countinvalid'],
    return_all=True,
    allow_failure=False,
)
```

Reply:
Fixed in 8f4fabf